ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -256,39 +233,6 @@ tests:
     - ref: ovn-sdn-migration-feature-migration-validate-multicast
     - ref: openshift-e2e-test
     workflow: openshift-e2e-aws-sdn
-- as: e2e-aws-sdn-network-migration-rollback
```

Comment:
So we're keeping sdn-to-ovn "old" migration, and sdn-to-ovn live migration, and sdn-to-ovn live migration rollback, but _not_ sdn-to-ovn "old" migration rollback? Any reason? It seems inconsistent...

Reply:
my intention was to remove as many jobs that do a fresh install with OpenShiftSDN. I missed removing
sdn-to-ovn "old" migration rollback. will update this PR for that.